packages/cli/src/init/test/lookup.ts

```diff
@@ -0,0 +1,253 @@
+import { values } from "@optique/core";
+import type { ChildProcessByStdio } from "node:child_process";
+import { spawn } from "node:child_process";
+import { createWriteStream } from "node:fs";
+import { join, sep } from "node:path";
+import process from "node:process";
+import type Stream from "node:stream";
+import { printErrorMessage, printMessage, runSubCommand } from "../../utils.ts";
+import { getDevCommand } from "../lib.ts";
+import type {
+  KvStore,
+  MessageQueue,
+  PackageManager,
+  WebFramework,
+} from "../types.ts";
+import webFrameworks from "../webframeworks.ts";
+
+const HANDLE = "john";
+const STARTUP_TIMEOUT = 30000; // 30 seconds
+const CWD = process.cwd();
+const BANNED_WFS: WebFramework[] = ["next"];
+
+/**
+ * Run servers for all generated apps and test them with the lookup command.
+ *
+ * @param dirs - Array of paths to generated app directories
+ */
+export default async function runServerAndReadUser(
+  dirs: string[],
+): Promise<void> {
+  const valid = dirs.filter(Boolean);
+  if (valid.length === 0) {
+    printErrorMessage`\nNo directories to lookup test.`;
+    return;
+  }
+  const filtered = filterWebFrameworks(valid);
+
+  printMessage`\nLookup Test start for ${String(filtered.length)} app(s)!`;
+
+  const results = await Array.fromAsync(filtered, testApp);
+
+  const successCount = results.filter(Boolean).length;
+  const failCount = results.length - successCount;
+
+  printMessage`Lookup Test Results:
+  Total: ${String(results.length)}
+  Passed: ${String(successCount)}
+  Failed: ${String(failCount)}\n\n`;
+}
+
+function filterWebFrameworks(
+  dirs: string[],
+): string[] {
+  const wfs = new Set<WebFramework>(
+    dirs.map((dir) => dir.split(sep).slice(-4, -3)[0] as WebFramework),
+  );
+  const hasBanned = BANNED_WFS.filter((wf) => wfs.has(wf));
+  if (!hasBanned) {
+    return dirs;
+  }
+  const bannedLabels = hasBanned.map((wf) => webFrameworks[wf]["label"]);
+  printErrorMessage`\n${
+    values(bannedLabels)
+  } is not supported in test mode yet.`;
+  return dirs.filter((dir) =>
+    !BANNED_WFS.includes(dir.split(sep).slice(-4, -3)[0] as WebFramework)
+  );
+}
+
+/**
+ * Run the dev server and test with lookup command.
+ */
+async function testApp(dir: string): Promise<boolean> {
+  const [wf, pm, kv, mq] = dir.split(sep).slice(-4) as //
+  [WebFramework, PackageManager, KvStore, MessageQueue];
+
+  printMessage`  Testing ${values([wf, pm, kv, mq])}...`;
+
+  const result = await serverClosure(
+    dir,
+    getDevCommand(pm),
+    sendLookup,
+  );
+
+  printMessage`    Lookup ${result ? "successful" : "failed"} for ${
+    values([wf, pm, kv, mq])
+  }!`;
+  if (!result) {
+    printMessage`    Check out these files for more details:
+      ${join(dir, "out.txt")}
+      ${join(dir, "err.txt")}\n`;
```

Comment:
```suggestion
    printMessage`    Check out these files for more details:
      ${join(dir, "out.txt")} and 
      ${join(dir, "err.txt")}\n`;
```

It's difficult to discern that error messages are being output to two different files. It would be better to include a word like `and` to explicitly distinguish them.